### DISCUSSION TITLE
2.1.0rc3

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#23435] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 2.1.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/2.1.0rc3/

Git tag for the release:
https://github.com/apache/superset/tree/2.1.0rc3

The Change Log for the release:
https://github.com/apache/superset/blob/2.1.0rc3/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/2.1.0rc3/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.0rc3'>2.1.0rc3</a>.</em>

**Comment by sfirke** [comment]
Looks great! 🙌   I tested generally as well as verified that the blocking issue (for me) from rc2 of email reports not working due to problems with the headless Firefox config are fixed.  The only regression I see is #23197 which is low-priority and can be mopped up at some point.

**Comment by IamJeffG** [comment]
We've gone so far as using it live, since 2.1.0rc3 resolves certain import-dashboard bugs that we had  on 2.0.1.  2.1.0rc3 is very stable.

I found https://github.com/apache/superset/issues/23456 to be a regression from 2.0.1, and while that feature is important for us, the new version is enough of an improvement to live with it for a while.

**Reply by eschutho** [reply]
Thanks for the testing and feedback, @IamJeffG!
